zilliqa/src/contracts/deposit.sol

```diff
@@ -273,6 +282,10 @@ contract Deposit {
         return committee().stakerKeys;
     }
 
+    function getTotalStake() public view returns (uint256) {
```

Comment:
Related but not part of this PR - I would suggest removing `totalStake` from `Committee`. The maximum number of stakes you need to add is only `256` and caching that result here just adds a potential inconsistency bug. In fact `consensus.rs` ignores the `totalStake` and calculates it for itself anyway.